app/src/main/java/de/westnordost/streetcomplete/map/MainFragment.kt

```diff
@@ -607,6 +611,25 @@ class MainFragment : Fragment(R.layout.fragment_main),
         mapFragment?.updateCameraPosition(300) { zoomBy = +1f }
     }
 
+    private fun onClickTracksStop() {
+
+        // hide the track information
+        binding.stopTracksButton.visibility = View.INVISIBLE
```

Comment:
Why not `View.GONE`?

Reply:
wasn't sure of the difference, changed it to GONE.